nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps

```diff
@@ -96,8 +96,8 @@ nb/ide.launcher/external/launcher-external-binaries-1-94a19f0.zip platform/o.n.b
 nb/ide.launcher/external/launcher-external-binaries-1-94a19f0.zip harness/apisupport.harness/external/launcher-external-binaries-1-94a19f0.zip
 
 # only one is part of the product:
-java/libs.javacapi/external/nb-javac-jdk-23+35-api.jar java/libs.nbjavacapi/external/nb-javac-jdk-23+35-api.jar
-java/libs.javacapi/external/nb-javac-jdk-23+35.jar java/libs.nbjavacapi/external/nb-javac-jdk-23+35.jar
+java/libs.javacapi/external/nb-javac-jdk-24+27-api.jar java/libs.nbjavacapi/external/nb-javac-jdk-24+27-api.jar
+java/libs.javacapi/external/nb-javac-jdk-24+27.jar java/libs.nbjavacapi/external/nb-javac-jdk-24+27.jar
```

Comment:
-> `24+29` to pass paperwork tests

Reply:
Right, fixed. I always forget about this.